packages/bot-web-ui/src/components/dashboard/tutorial-tab/quick-strategy-content/quick-strategy-cards.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Icon, Text } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import { STRATEGIES } from 'Components/quick-strategy/config';
+import QuickStrategyGuidesDetail from './quick-strategy-guides-details';
+
+type Tcontent = {
+    qs_name: string;
+    content: string[];
+    type: string;
+};
+
+type TQuickStrategyGuides = {
+    quick_strategy_content: Tcontent[];
+    tutorial_selected_strategy: string;
+    setTutorialSelectedStrategy: (value: string) => void;
+};
+
+const QuickStrategyCards = observer(
+    ({ quick_strategy_content, tutorial_selected_strategy, setTutorialSelectedStrategy }: TQuickStrategyGuides) => {
+        const { ui } = useStore();
+        const { is_mobile } = ui;
+
+        return (
+            <>
+                {tutorial_selected_strategy === '' ? (
+                    <div className='tutorials-quick-strategy__cards'>
+                        {quick_strategy_content?.map(({ qs_name, content, type }) => (
+                            <div
+                                className='tutorials-quick-strategy__placeholder'
+                                key={type}
+                                onClick={() => setTutorialSelectedStrategy(qs_name)}
+                                role='button'
+                                tabIndex={0}
+                            >
+                                <div>
+                                    <div className='tutorials-quick-strategy__placeholder__title'>
+                                        <Text
+                                            align='center'
+                                            weight='bold'
+                                            color='prominent'
+                                            line_height='s'
+                                            size={is_mobile ? 'xxs' : 's'}
```

Comment:
maybe it would be better to use one variable for this expression: 
`is_mobile ? 'xxs' : 's'` instead 4 time to repeat it, wouldn't it?

Reply:
sure ✅